etna/transforms/feature_selection/feature_importance.py

```diff
@@ -94,3 +98,66 @@ def fit(self, df: pd.DataFrame) -> "TreeFeatureSelectionTransform":
         weights = self._get_regressors_weights(df)
         self.selected_regressors = self._select_top_k_regressors(weights, self.top_k)
         return self
+
+
+class MRMRFeatureSelectionTransform(BaseFeatureSelectionTransform):
+    """Transform that selects regressors according to MRMR variable selection method."""
```

Comment:
This is not quite a mrmr method, but a slightly modified one. Maybe so and write that "modified mrmr" so that no one gets confused?

Reply:
What do you mean under "modifications"?